src/engine/Sim3D.ts

```diff
@@ -358,32 +358,44 @@ export class Sim3D {
 
     if (worldConfig.walls) {
       // We want walls
-      if (worldConfig.walls.length === 0) {
-        // // Empty array, generate the default set of walls (perimeter)
+      if (!worldConfig.walls.walls) {
+        // // Empty array, generate the default set of walls (perimeter) with the given height
+        const height = worldConfig.walls.height ? worldConfig.walls.height : 1;
```

Comment:
Can we make the default height and thickness a constant and define it at the top of the `Sim3D` file?

Reply:
yeah like i said, these constants are already defined in SimWall, so I wasn't sure if you wanted to put them in some common place, or access them from SimWall in Sim3D. That's why I left them like this for now.

Reply:
Ah right. OK, so in `SimWall` you can add `export` in front of `const DEFAULT_WALL_THICKNESS ....` and `const DEFAULT_WALL_HEIGHT ...`, and this will let you access them in `Sim3D`.